assets/src/block-editor/helpers/index.js

```diff
@@ -344,8 +344,13 @@ ImageBlockLayoutAttributes.propTypes = {
  */
 const GalleryBlockLayoutAttributes = (props) => (
 	<InspectorAdvancedControls>
-		<AmpLightboxToggle {...props} />
-		<AmpCarouselToggle {...props} />
+		<div>
+			<p className="components-base-control__label">
```

Comment:
I guess there's no component specifically for this?

Reply:
Unable to find such on https://wordpress.github.io/gutenberg/.

Reply:
Tried `<Placeholder />` but it creates a border with a box and doesn't fit. See working at https://wordpress.github.io/gutenberg/?path=/story/components-placeholder--default

Reply:
What about `BaseControl.VisualLabel`? 

https://github.com/WordPress/gutenberg/blob/64a873a80b001a1c2c892c0c4a56010a94cc4c03/packages/components/src/base-control/index.tsx#L101-L135
https://wordpress.github.io/gutenberg/?path=/story/components-basecontrol--with-visual-label

Reply:
Maybe that has to be inside of a `BaseControl`?

Reply:
Yes, it works. Thanks 🙌🏼 

![image](https://user-images.githubusercontent.com/54371619/217096016-71baa48c-1ed1-470d-bafb-e6dab6c57db1.png)